### DISCUSSION TITLE
Welcome to UnityFigmaBridge Discussions!

### DISCUSSION THREAD

**Discussion by simonoliver** [simonoliver/UnityFigmaBridge#9] [Announcements]
## 👋 Welcome!
I'm Simon - original author of Unity Figma Bridge and found of indie studio @HandCircus. I’ve just added Discussions as a place to connect with other users of UnityFigmaBridge. I'd be really curious to know more about how you use Figma or the tool, any suggestions on improvements and any issues you have. And feel free to ask any questions.

Introductions very welcome too!

**Comment by AtomicRogue1** [comment]
Hey, I tried using your utility for implementing UI from this figma file: https://www.figma.com/file/0OoSOuXhFFdfv8MTYMFJJp
I followed your steps as suggested in the README.md. However, when I sync my document after inputting my access token, it downloads files but then stops midway to show this error:
![image](https://user-images.githubusercontent.com/30554253/233835213-c1ce3b8c-8c26-465e-a213-9a818d9e4d07.png)
![image](https://user-images.githubusercontent.com/30554253/233835256-ca472085-cf56-4750-ad06-69ffb46d0557.png)
Note that this error repeats regardless of me using the best wifi. Just wanted to report this error that's all if a fix is possible or please tell if I am doing something wrong.

**Reply by simonoliver** [reply]
Hey thanks for raising this and for sharing the link. Looks like it's failing rendering the stand in images - as it's quite a big file maybe there are too many for one batch? I'll take a look and see if I can make a fix

**Reply by simonoliver** [reply]
Ok taking a look at this - the issue seems to be that the server is trying to render images that are too large. If you change "server render image scale" in the settings file to 1, that part will work again. It does get a lot further but does hit an issue on the "item spritesheets" page that I'll try and fix up in the next couple of weeks.

It takes a long time though! This is a great project to use for optimization, thanks for sharing.

![image](https://user-images.githubusercontent.com/1059515/233982070-a08ec9d4-1000-4fd2-8e28-3772f3bd4ddf.png)
